biit_server/account.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any, Dict
+
+
+class Account:
+    def __init__(
+        self,
+        email=None,
+        fname=None,
+        lname=None,
+        age=20,
+        agePref="",
+        birthday="",
+        covid=None,
+        meetLength=None,
+        meetType=None,
+        optIn=None,
+        schedule=None,
+        document_snapshot=None,
+    ):
+        if document_snapshot != None:
+            data = document_snapshot.to_dict()
+            self.email = data.get("email")
+            self.fname = data.get("fname")
+            self.lname = data.get("lname")
+            self.age = data.get("age")
+            self.agePref = data.get("agePref")
+            self.birthday = data.get("birthday")
+            self.covid = data.get("covid")
+            self.meetLength = data.get("covid")
+            self.meetType = data.get("covid")
+            self.optIn = data.get("optIn", 0)
+            self.schedule = data.get("schedule")
+            return
+
+        self.email = email
+        self.fname = fname
+        self.lname = lname
+        self.age = age
+        self.agePref = agePref
+        self.birthday = birthday
+        self.covid = covid
+        self.meetLength = meetLength
+        self.meetType = meetType
+        self.optIn = (optIn,)
+        self.schedule = schedule
+
+    def set_property(self, property, value):
+        value = self.__dict__.get("property")
+
+        if value != None:
+            self.__dict__[property] = value
```

Comment:
if we are going to block `None` values we should log something when `None` is passed to it. will make debugging easier